sklego/preprocessing.py

```diff
@@ -200,6 +201,9 @@ def transform(self, X):
             return X[self.columns]
         return X
 
+    def get_feature_names(self):
```

Comment:
seems fair, but i wonder ... any reason why this cannot be a property?

Reply:
I agree that a property would be prettier, but scikit-learn does it like this in several other places as well (https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html#sklearn.feature_extraction.text.TfidfVectorizer.get_feature_names)

Reply:
Fair enough. :)